raiden/tests/unit/test_matrix_transport.py

```diff
@@ -24,31 +22,6 @@
 from raiden.utils.signer import recover
 
 
-def test_join_broadcast_room():
-    """ join_broadcast_room should try joining, fail and then create broadcast public room """
```

Comment:
This PR changes that function so that it wont create the room anymore, the test is not useful anymore.